#!/bin/sh /usr/share/dpatch/dpatch-run
## 31-magic-add-ape.dpatch by Robert Millan <rmh@aybabtu.com>
##
## DP: New magic for the Monkey's audio format (Closes: #334862).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/audio file-4.20/magic/Magdir/audio
--- file-4.20.orig/magic/Magdir/audio	2007-01-19 19:26:35.000000000 +0000
+++ file-4.20/magic/Magdir/audio	2007-03-27 16:22:03.000000000 +0000
@@ -552,3 +552,7 @@
 # http://download.ncl.ie/doc/api/ie/ncl/media/music/IMelody.html
 # http://www.wx800.com/msg/download/irda/iMelody.pdf
 0	string	BEGIN:IMELODY	iMelody Ringtone Format
+
+# From: Robert Millan <rmh@aybabtu.com>
+# Monkey's Audio <http://www.monkeysaudio.com/>
+0	string	MAC\		Monkey's Audio file
